octobot_trading/exchanges/implementations/margin_ccxt_exchange.py

```diff
@@ -21,18 +21,44 @@
 import octobot_trading.exchanges.types as exchanges_types
 import octobot_trading.personal_data as personal_data
 from octobot_trading import enums as enums
+from octobot_trading.exchanges.util import parser
 
 
 #TODO remove
 class MarginCCXTExchange(exchanges_types.MarginExchange):
     CONNECTOR_CLASS = exchange_connectors.CCXTExchange
-
+    
+    # overwrite classes if you need a different parser
+    ORDERS_PARSER_CLASS = parser.OrdersParser
+    TRADES_PARSER_CLASS = parser.TradesParser
+    POSITIONS_PARSER_CLASS = parser.PositionsParser
+    HAS_STOP_ENDPOINT = True # check should be good tho
+
+    CANDLE_LOADING_LIMIT = 0
+    MAX_RECENT_TRADES_PAGINATION_LIMIT = 0
+    MAX_ORDER_PAGINATION_LIMIT = 0
+
+    USE_FIXED_MARKET_STATUS = None  # None -> default is with fixed market status
+    #                                 False -> define False if not necessary
+    #                                 True -> set to True if exchange is know to need fixed market status
+
+    MARKET_STATUS_FIXER_REMOVE_PRICE_LIMITS = False
+    
     def __init__(self, config, exchange_manager):
         super().__init__(config, exchange_manager)
         self.connector = self.CONNECTOR_CLASS(
             config,
             exchange_manager,
-            additional_ccxt_config=self.get_additional_connector_config()
+            additional_ccxt_config=self.get_additional_connector_config(),
+            candle_loading_limit=self.CANDLE_LOADING_LIMIT,
+            max_order_pagination_limit=self.MAX_ORDER_PAGINATION_LIMIT,
+            max_recent_trades_pagination_limit=self.MAX_RECENT_TRADES_PAGINATION_LIMIT,
+            use_fixed_market_status=self.USE_FIXED_MARKET_STATUS,
+            market_status_fixer_remove_price_limits=self.MARKET_STATUS_FIXER_REMOVE_PRICE_LIMITS,           
+            orders_parser=self.ORDERS_PARSER_CLASS,
+            trades_parser=self.TRADES_PARSER_CLASS,
+            positions_parser=self.POSITIONS_PARSER_CLASS,
+            HAS_STOP_ENDPOINT=self.HAS_STOP_ENDPOINT,
```

Comment:
What will it be used for?

Reply:
HAS_STOP_ENDPOINT is removed, the parsers are set here, so you can override them from the exchange tentacle (see bybit)